### PR TITLE
Marquee revised

### PR DESCRIPTION
**Pull request by pulkitSoni526**
refactor marquee script

### CODE REVIEW

File: tools/importer/import.js [hunk 1]
```diff
@@ -34,6 +34,8 @@ import { rgbToHex } from './utils.js';
 
 import fetchBlockScript from './fetchBlockScript.js';
 
+// import createCardsBlock from './rules/creativecloud/file-types/cards.js';
```
Comment (saurabhsircar11) [inline tools/importer/import.js]: Remove commented code

File: tools/importer/rules/longText.js [hunk 2]
```diff
@@ -1,7 +1,24 @@
 /* global WebImporter */
+// export default function longText(block, document) {
```
Comment (saurabhsircar11) [inline tools/importer/rules/longText.js]: Remove commented code

File: tools/importer/rules/marquee.js [hunk 3]
```diff
@@ -1,142 +1,233 @@
...
-      if (imgSrc.indexOf('https') === -1) {
-        imgSrc = `https://www.adobe.com/${imgSrc}`;
-      }
-      const imgLink = document.createElement('a');
-      imgLink.innerHTML = imgSrc;
-      imgLink.setAttribute('href', imgSrc);
-      textElement.appendChild(imgLink);
-    }
+/**
+ * Function for building Marquee,
```
Comment (saurabhsircar11) [inline tools/importer/rules/marquee.js]: JsDoc is not attached to the right function, it is for createMarquee function. Please put this above it

File: tools/importer/import.js [hunk 4]
```diff
@@ -253,6 +255,7 @@ export default {
       const divOffset = parseInt(id.split('-').pop(), 10);
       createBlocks(name, divOffset);
     });
+    createMarqueeBlocks((document.querySelector('#root_content_flex_copy_copy_copyu')), document);
```
Comment (saurabhsircar11) [inline tools/importer/import.js]: Please remove this

File: tools/importer/import.js (hunk 1 above)
Comment (pulkitSoni526) [reply tools/importer/import.js]: done

File: tools/importer/rules/longText.js (hunk 2 above)
Comment (pulkitSoni526) [reply tools/importer/rules/longText.js]: done

File: tools/importer/rules/marquee.js (hunk 3 above)
Comment (pulkitSoni526) [reply tools/importer/rules/marquee.js]: done

File: tools/importer/import.js (hunk 4 above)
Comment (pulkitSoni526) [reply tools/importer/import.js]: done

**Review by Deva309 — APPROVED** [review]

**Pull request merged**